core/src/main/resources/com/squarespace/template/plugins/platform/quantity-input.html

```diff
@@ -1,6 +1,6 @@
 {.var @quantityLabel localizedStrings.productQuantityInputLabel}
 
 <div class="product-quantity-input" data-item-id="{id}" data-animation-role="content">
-  <div class="quantity-label">{.if @quantityLabel}{@quantityLabel}{.or}Quantity{.end}:</div>
-  <input size="4" max="9999" min="1" value="1" type="number" step="1"></input>
+  <label for="product-quantity" class="quantity-label">{.if @quantityLabel}{@quantityLabel}{.or}Quantity{.end}:</label>
+  <input id="product-quantity" size="4" max="9999" min="1" value="1" type="number" step="1"></input>
```

Comment:
Do we want to make this a bit less generic? Something like `sqs-product-quantity`? I'm thinking about the edge case of existing sites having custom code where some element already has an id of `product-quantity`

Reply:
sure. i figured `product-quantity` should be ok, but prepending sqs is a good call